tools/api-markdown-documenter/src/api-item-transforms/helpers/TableHelpers.ts

```diff
@@ -418,8 +428,13 @@ export function createPropertiesTable(
 		return undefined;
 	}
 
-	// Only display "Alerts" column if there are any deprecated items in the list.
+	// Only display "Alerts" column if there are any deprecated or alpha/beta items in the list.
 	const hasDeprecated = apiProperties.some((element) => isDeprecated(element));
+	const hasAlphaOrBeta = apiProperties.some((element) => {
+		const releaseTag = getReleaseTag(element);
+		return releaseTag === ReleaseTag.Alpha || releaseTag === ReleaseTag.Beta;
+	});
+	const hasAlerts = hasDeprecated || hasAlphaOrBeta;
```

Comment:
Might be worth making this a helper, I don't think the first two are used at all?

Reply:
Yeah, that's probably a good call. Not all of the tables use this pattern, but enough do.